.github/workflows/continuous-integration-workflow.yml

```diff
@@ -88,6 +88,20 @@ jobs:
           name: aioxmpp test output
           path: aioxmpp/output
 
+  check_branch:
+    runs-on: ubuntu-latest
+    outputs:
+      is_publishable_branch: ${{ steps.check-branch.outputs.is_publishable_branch }}
+    steps:
+      - name: check branch ${{ github.ref }} is either master or a version number
+        id: check-branch
+        run: |
+          if [[${{ github.ref }} == 'refs/heads/master' || ${{ github.ref }} =~ refs\/heads\/[0-9]+\.[0-9]+ ]]; then
```

Comment:
Double quotes around the `${{ github.ref }}`s?

Reply:
If its BASH, then it need a space between `[[` and the expression.

Reply:
And double quotes around will make it save against spaces in the path. But there may be probably other things that will break in this case :smile:

Reply:
Thanks, is this now resolved?